roles/netbootxyz/templates/menu/smartos.ipxe.j2

```diff
@@ -28,7 +28,7 @@ item toggle_kmdb_b ${space} Boot Kernel Debugger First: ${kmdb_b}
 
 iseq ${noimport} true && item --gap ${space} ||
 iseq ${noimport} true && item --gap ${space} Zpool will not be imported. Rescue mode root password can be found at ||
-iseq ${noimport} true && item --gap ${space} http://us-east.manta.joyent.com/Joyent_Dev/public/SmartOS/smartos.html ||
+iseq ${noimport} true && item --gap ${space} http://us-central.manta.mnx.io/Joyent_Dev/public/SmartOS/smartos.html ||
```

Comment:
This seems to provide invalid request, but it works with https, should we change it to that?

Reply:
@antonym It works for me...?

```
] curl -s -i http://us-central.manta.mnx.io/Joyent_Dev/public/SmartOS/smartos.html | head -20
HTTP/1.1 200 OK
server: Manta/2
etag: 69dcbb1b-f49b-4bf5-8a94-8dce4d28cfa0
last-modified: Fri, 29 Jul 2022 16:13:27 GMT
access-control-allow-origin: *
accept-ranges: bytes
content-type: text/html
content-md5: 6w9BbucI3Jxpmaso7QQLFA==
content-length: 20172
durability-level: 2
date: Sun, 14 Jan 2024 19:31:36 GMT
x-request-id: 75dd0fad-be3b-4f5c-8cb9-e18e6d43def7
x-response-time: 139
x-server-name: 6492ecf3-666a-460f-b4ac-072984eb8c8d

<!doctype html>
<!--
SmartOS Changelog made pretty
```

Are you using HTTP/2.0 by any chance?

But, now that you mention it, that URL is for end users and not for ipxe itself, so there's no reason for it to not be https.